trieste/acquisition/optimizer.py

```diff
@@ -98,32 +100,54 @@ def automatic_optimizer_selector(
 
     else:
         raise NotImplementedError(
-            f""" No optimizer currentely supports acquisition function
+            f""" No optimizer currently supports acquisition function
                     maximisation over search spaces of type {space}.
                     Try specifying the optimize_random optimizer"""
         )
 
 
-def optimize_discrete(space: DiscreteSearchSpace, target_func: AcquisitionFunction) -> TensorType:
+def optimize_discrete(
+    space: DiscreteSearchSpace,
+    target_func: Union[AcquisitionFunction, Tuple[AcquisitionFunction, int]],
+) -> TensorType:
     """
     An :const:`AcquisitionOptimizer` for :class:'DiscreteSearchSpace' spaces and
     batches of size of 1.
 
+    This function also support the maximization of vectorized target functions by evaluating
+    the points in the search space for each of the individual V functions making up target_func.
+
```

Comment:
it would be great to add a brief sentence on when are these vectorized target functions used